minrx.cpp

```diff
@@ -39,6 +39,19 @@
 
 #include "minrx.h"
 
+#ifdef HAVE_CONFIG_H
```

Comment:
I'm curious why angle brackets for <config.h> and quotes for "gettext.h"?  This is almost exactly the opposite of what I would have expected: config.h is clearly a local header file to whatever project you're building, and gettext.h is quite often in /usr/include on modern systems.

Reply:
I copied this from gawk.  Gawk has its own version of gettext.h.  However, you're right, for gettext.h it should be angle brackets and I've made that change.

For config.h it's less clear. I have a vague memory that the Autoconf doc wants it to be in angle brackets but I don't remember why, so I've gone ahead and changed it to double quotes. We can change it back later if needed.

I've also fixed `minrx_reg_error()` to call `gettext()` on the error string.  Thanks.